lib/services/android-debug-service.ts

```diff
@@ -54,10 +54,13 @@ class AndroidDebugService implements IDebugService {
 	public debugOnDevice(): IFuture<void> {
 		return (() => {
 			let packageFile = "";
+            
+            if(!this.$options.debugBrk && !this.$options.start) {
+                this.$logger.warn("Neither --debug-brk nor --start option was specified. Defaulting to --debug-brk.");                
+                this.$options.debugBrk = true;
+            }
 
 			if (this.$options.debugBrk) {
-				this.$platformService.preparePlatform(this.platform).wait();
```

Comment:
because prepare function is called from build - https://github.com/NativeScript/nativescript-cli/pull/740/files#diff-c8a25497a509cafa1678ac3583ceb7e5R66. This way the project was prepared 2 times :smile_cat: